modules/standard/Math.chpl

```diff
@@ -473,24 +473,28 @@ module Math {
     return chpl_divfloorpos(x, y);
   }
 
-  /* Returns the error function of the argument `x`. */
+  /* Returns the error function of the argument `x`. This is equivalent to the
+     integral of :proc:`twiceReciprSqrtPi`\ * :proc:`exp`\(`-t**2`)dt from 0
```

Comment:
If Wikipedia is to be believed, I think it should be: "This is equivalent to :param:`twiceReciprSqrtPi` * the integral of :proc:`exp`\(`-t**2`)dt from 0 to `x`"

Reply:
Well spotted on the use of the param and function, though, I think I would have missed referencing that when I got around to it :)

Reply:
Actually, `twiceReciprSqrtPi` is a constant so it can be placed inside or outside the integral without any effect on the result. I chose to put it inside because I wasn't convince by the notation of the form: "maths * text followed by some other maths". But you're right, this is the most commonly known form.

Ok, I'm going to remove these references to make things simple.